docs/source/migrations/1.0.mdx

```diff
@@ -6,69 +6,69 @@ description: From 0.x to 1.0
+- Code generation tooling written in pure Swift code.
+- Generated models that improve readability and functionality while reducing generated code size.
+- Support for using generated models in multi-module projects.
+- Type-safe APIs for cache key resolution.
 
-> **To start your migration to Apollo iOS 1.0:**
-> - You should first read about the [key changes](#key-changes) highlighted below.
-> - Then, follow the [step-by-step instructions](#step-by-step-instructions).
-> - Lastly, use the [breaking changes](#breaking-changes) section to help you resolve any remaining build errors in your code.
+This article describes the significant changes for this major version of Apollo iOS and a walk-through of migrating an existing project from Apollo iOS 0.x to 1.0.
+
+> **Steps to migrate to Apollo iOS 1.0:**
+> 1. Begin by reading about the [key changes](#key-changes).
+> 2. Next, follow our [step-by-step instructions](#step-by-step-instructions) to upgrade your existing app.
+> 3. Finally, see [breaking changes](#breaking-changes) to resolve any remaining build errors.
 
 ## Key changes
 
 ### Generated schema module
 
-In addition to generating models for your GraphQL operation definitions, the 0.x version generated models for the input objects and enums referenced from your schema.
+The 0.x version of Apollo iOS generates models for your GraphQL operation definitions _and_ the input objects and enums referenced in your schema.
+
+Apollo iOS 1.0 expands on this, generating an _entire module_ that contains models and metadata for your GraphQL schema and its type definitions.
 
-Apollo iOS 1.0 expands on this, generating a whole module that contains models and metadata for your GraphQL schema and its type definitions.
 In addition to your input objects and enum types, this module contains:
-- Types for the objects, interfaces, and unions referenced
-- Editable definitions for your custom scalars
-- An editable `SchemaConfiguration.swift` file
-- Metadata used by the Apollo GraphQL executor
+- Types for the objects, interfaces, and unions in your schema.
+- Editable definitions for your custom scalars.
+- An editable `SchemaConfiguration.swift` file.
+- Metadata for the Apollo GraphQL executor.
 
-To prevent naming conflicts with other types in your project, the schema types are contained in their own namespace.
-This namespace can be generated as a stand-alone module, which can be imported by your project, or as a caseless namespace enum which can be embedded in your application target.
+Schema types are contained in their own namespace to prevent naming conflicts. You can generate this namespace as a stand-alone module, imported by your project, or as a caseless namespace enum that you can embed in your target application.
```

Comment:
```suggestion
Schema types are contained in their own namespace to prevent naming conflicts. You can generate this namespace as a stand-alone module, imported by your project, or as a caseless namespace enum that you can embed in your application target.
```